docs/diagrams/UiClassDiagram.puml

```diff
@@ -32,29 +34,26 @@ UiManager .left.|> Ui
 UiManager -down-> "1" MainWindow
 MainWindow *-down->  "1" CommandBox
 MainWindow *-down-> "1" ResultDisplay
+MainWindow *-down-> "1" FocusCard
 MainWindow *-down-> "1" CandidateListPanel
 MainWindow *-down-> "1" StatusBarFooter
+MainWindow *-down-> "1" InterviewListPanel
 MainWindow --> "0..1" HelpWindow
 
 CandidateListPanel -down-> "*" CandidateCard
```

Comment:
This bugger `*` multiplicity is too close to the arrow but I tried to fix it :')